src/mono/System.Private.CoreLib/src/System/Reflection/RuntimeFieldInfo.cs

```diff
@@ -217,7 +214,7 @@ public override object[] GetCustomAttributes(Type attributeType, bool inherit)
 
         public override string ToString()
         {
-            return string.Format("{0} {1}", FieldType, name);
+            return SR.Format("{0} {1}", FieldType, name);
```

Comment:
Since this case doesn't use a resource string, it should be `string.Format`.

```suggestion
            return string.Format("{0} {1}", FieldType, name);
```